src/plugins/data/common/es_query/kuery/functions/range.ts

```diff
@@ -24,14 +24,15 @@ import { getRangeScript } from '../../filters';
 import { getFields } from './utils/get_fields';
 import { getTimeZoneFromSettings } from '../../utils';
 import { getFullFieldNameNode } from './utils/get_full_field_name_node';
+import { IIndexPattern, KueryNode, IFieldType } from '../../..';
 
-export function buildNodeParams(fieldName, params) {
+export function buildNodeParams(fieldName: any, params: any) {
```

Comment:
I think `params` can be typed to `RangeFilterParams` here, which would get rid of several of the `any`s in this file.